src/lib/prompts/quick-pick.ts

```diff
@@ -7,30 +7,34 @@ import * as vscode from 'vscode';
 export default function createQuickPick<T extends vscode.QuickPickItem>({
   placeholder,
   items = [],
-  format = (i) => i,
+  value = '',
   step,
   totalSteps,
-}: Partial<
-  vscode.QuickPick<T> & {
-    format(i: T[]): T[];
-  }
->): Promise<T[]> {
-  return new Promise(function (resolve) {
+  buttons = [],
+}: Partial<vscode.QuickPick<T>>): Promise<T[]> {
+  return new Promise(function (resolve, reject) {
     const picker = vscode.window.createQuickPick();
     picker.placeholder = placeholder;
     picker.matchOnDescription = true;
     picker.matchOnDetail = true;
     picker.ignoreFocusOut = true;
     picker.items = items;
+    picker.value = value;
```

Comment:
> In the case of `QuickPick`, it is possible that `selectedItems` would be preferable to `value` as a default when we are back, but I haven't done so because it would require major code changes to do so.

```suggestion
    picker.value = '';
```

Make the value of the picker always be `''` will make `selectedItems` always be displayed. 😁

Reply:
Yes, but if you don't assign the previous `selectedItems` to `selectedItems`, the previous selection will not be retained.

Reply:
@toddlerer I don't understand what process you want 😅. Below I describe the process I want:

1. There are three quick selection dialogs. If we don't consider the scope item (its options are more complicated), the type item and gitmoji item are the main concerns. The return from description is taken as an example later.
2. If we return from the description, it will return to the gitmoji item. Modify the selection at this time. If we continue, the content of the description will still be retained, and the previous type and scope will also be retained.
3. If we return from description to type, then the scope and gitmoji need to be reselected, the content of description will still be retained, and the final content will be the result of the second selection.

After my local test, modifying the initial value here will make the value of the quick selection dialog empty, which will not affect subsequent modifications.

If you want the previously selected option to remain as the default option, you may need to change the order of the options. 🧐

Reply:
I don't have a problem with it being retained in the case of `InputBox`, but the problem is the retention of items in the case of `QuickPick` and `ConfigurableQuickPick`.
Sorry, I must have misread the API and was thinking of `selectedItems` as `activeItems`.
It turns out that I can retain the items by assigning the previous selection to `activeItems` without reordering the items.

It turns out that it doesn't require that much of a change, so I'll fix it within the next few days.